QuizApp.DAL/Repository/Contracts/IRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
```

Comment:
The same issue with branches. Please target the current branch to the branch with repositories, not to the master.